ibr/ibr-sdk/src/renderers.js

```diff
@@ -472,10 +475,18 @@ function triangulatePoints(poly) {
  * Serialize IBRObject object to binary format.
  * @param {IBRObject} ibrObject The IBRObject to be saved back to binary
  format.
+ * @param {List.<Boolean>} floorToSave Boolean array representing the floors
+ to be exported. If element at index i is true, the i-th floor will be
+ exported.
  * @return {Buffer} buffer binary representation of IBRObject object.
  */
-function saveToBuffer(ibrObject) {
+function saveToBuffer(ibrObject, floorToSave) {
   const json = ibrObject.toJson();
+  for (const i in floorToSave) {
```

Comment:
docstring says floorToSave is list of booleans? also nit: s/floorToSave/floorsToSave/g if doing multiple floors. also is i a boolean or int?